costmodel/costmodel.go

```diff
@@ -2023,6 +2044,10 @@ func (cm *CostModel) costDataRange(cli prometheusClient.Client, clientset kubern
 	}
 	for key := range GPUReqMap {
 		containers[key] = true
+		klog.Infof("GPUMAP: %s", key)
```

Comment:
Log level for this?

Reply:
Oops. Accidentally left these in for debug. Thanks